LICENSE

```diff
@@ -19,7 +19,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 This program incorporates work covered by the following copyright and
 permission notices:
 
-  WordPress - Web publishing software, and specificially the Gutenberg
```

Comment:
Ha! This looks to have started in https://github.com/happyprime/content-aggregator-block/commit/2f1c3d5df80374a659b3ac36554992ad5d8cee57